test/utility-test.js

```diff
@@ -107,108 +107,6 @@ describe('Database Util', function() {
       });
     });
   });
-  describe('#getRowById()', function() {
```

Comment:
Should we keep these tests? Is the worker going to be using at least `getRowById`?